cips/cip-6-second-blocks.md

```diff
@@ -0,0 +1,60 @@
+| cip | TBD |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
+| author | Josh Stein ([@jcstein](https://github.com/jcstein)) |
+| discussions-to | <https://forum.celestia.org/t/cip-decrease-block-time-to-6-seconds/1836> |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-10-09 |
+
+## Abstract
+
+This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will increase the throughput by double and and reduce the time it takes for transactions to be finalized by half.
+
+## Motivation
+
+The motivation for this CIP stems from a discussion in Core Devs Call 17, where it was proposed to reduce the block time to 6 seconds from 12 seconds.
+
+## Specification
+
+1. The block time in celestia-app MUST be reduced from 12 seconds to 6 seconds.
+
+2. All implementations of celestia-app SHALL adjust their block production mechanisms to conform to this new 6-second block time.
+
+3. The change in block time MUST be implemented at a specific block height, which SHALL be determined and agreed upon by the Celestia community.
+
+4. Celestia consensus nodes SHOULD update their software to accommodate this change prior to the agreed-upon block height.
+
+5. The reduction in block time SHALL NOT affect the overall security or integrity of the Celestia network.
+
+6. Validators MUST ensure that their systems are capable of producing and validating blocks at this increased rate.
```

Comment:
I don't think we have guidance on how validators should ensure their systems are capable of doing this. IMO if they upgrade to the correct version, then they should be fine. 

```suggestion
```